pkg/handler/v2/routes_v2.go

```diff
@@ -6302,6 +6306,31 @@ func (r Routing) listEKSRegions() http.Handler {
 	)
 }
 
+// swagger:route GET /api/v2/providers/eks/clusterroles eks listEKSClusterRoles
+//
+//	List EKS Cluster Service Roles.
+//
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: []EKSClusterRolesList
```

Comment:
Shouldn't it be `EKSClusterRolesList` instead of `[]EKSClusterRolesList` here? Because `EKSClusterRolesList` is already a slice of string and it's the type being returned.